spec/models/claim/subject_check_email_data_export_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+require "csv"
+
+RSpec.describe Claim::SubjectCheckEmailDataExport do
+  let(:submitted_student_loans_claims) do
+    Array.new(5) do
+      eligibility = create(:student_loans_eligibility, :eligible, biology_taught: true, chemistry_taught: true, computing_taught: false, languages_taught: false, physics_taught: true)
+      create(:claim, :submitted, eligibility: eligibility, first_name: "John", middle_name: "Herbert", surname: "Adams")
+    end
+  end
+  let(:submitted_maths_and_physics_claims) { create_list(:claim, 5, :submitted, policy: MathsAndPhysics) }
+  let!(:submitted_claims) { submitted_student_loans_claims + submitted_maths_and_physics_claims }
+  let!(:submittable_claims) { create_list(:claim, 4, :submittable) }
+  let!(:excluded_claims) { create_list(:claim, 3, :submitted) }
+
+  subject { described_class.new(excluded_claims.map(&:reference).join(",")) }
+
+  describe "#csv_string" do
+    let(:csv) { CSV.parse(subject.csv_string, headers: true) }
+
+    it "returns a parseable CSV string with the expected headers" do
+      expect(csv.headers).to eq(["Claim reference", "Policy", "Current school URN", "Current school name", "Claimant name", "Subject"])
+    end
+
+    it "contains a row for each submitted, non-excluded claim" do
+      expect(csv.map { |row| row["Claim reference"] }).to match_array(submitted_claims.map(&:reference))
+    end
+
+    it "includes claims’ reference, policy, current school URN, current school name, and claimant name excluding middle name" do
+      claim = submitted_student_loans_claims.first
```

Comment:
For a spot of  extra robustness:

```suggestion
      claim = submitted_student_loans_claims.sample
```

Reply:
I think this is no longer needed now that we're only using one claim.